next.config.js

```diff
@@ -28,13 +30,20 @@ module.exports = {
     const aliases = config.resolve.alias || (config.resolve.alias = {});
     aliases.react = aliases['react-dom'] = 'preact/compat';
 
-    // inject Preact DevTools
     if (dev && !isServer) {
       const entry = config.entry;
+      // inject Preact DevTools
       config.entry = () => entry().then(entries => {
         entries['main.js'] = ['preact/debug'].concat(entries['main.js'] || []);
         return entries;
       });
+
+      // Inject Prefresh
+      const reactRefresh = config.plugins.find(s => s.constructor.name === 'ReactFreshWebpackPlugin');
+      if (reactRefresh) {
+        config.plugins.splice(config.plugins.indexOf(reactRefresh), 1);
+      }
```

Comment:
Ideally we'd have an else branch here to add this when we are at pre 9.4 but couldn't figure out how